contracts/Alchemica/AlchemicaToken.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+

```

Comment:
Should specify a specific version of Solidity here.  I recommend using the latest production version of Solidity for all new code. The current version is Solidity 0.8.13

Reply:
Updated solidity version.